MM-27688 - Implement OpenId Connect in Cloud (#4303)
mattermost/docs: source/cloud/cloud-administration/converting-oauth-2.0-to-openid-connect.rst

Converting OAuth 2.0 to OpenID Connect
=======================================

> OAuth 2.0 is being deprecated and replaced by OpenID Connect. You need to migrate your OAuth configuration to the new OpenID Connect standard. 

Author: Removed "new" from the 3 supported service provider topics and the conversion topic.

> OAuth 2.0 is being deprecated in the System Console and replaced by OpenID Connect. Using the System Console, you can migrate your OAuth configuration to the OpenID Connect standard. 

Reviewer: ```suggestion
OAuth 2.0 is being deprecated and replaced by OpenID Connect. Using the System Console, you can migrate your OAuth configuration to the OpenID Connect standard. 
```

Author: @justinegeffen In this specific case, OAuth 2.0 is being deprecated only in the UI (the System Console). System Admins aren't being forced to migrate their existing OAuth 2.0 configurations to OpenID Connect (though we give them a quick way to do so). Given this context, would you continue to recommend that we remove the reference to the System Console here?

Reviewer: Thanks for the context, and no, agreed that this would not work.

> [within what timeframe?] [failure to do so will result in....tbd]

Author: @wiersgallak @sbishel Can you provide some details about the urgency around converting, please? Is there a deadline that should be adhered to? Are there consequences to System Admins not converting ASAP?

Author: Thanks! I've adjusted the tone of the sentence to feel less urgent/time sensitive.

.. image:: ../../images/OAuth20Deprecated.png

> A one-time, one-click conversion tool is available within the existing **OAuth 2.0** page or within the new **OpenID Connect** page by going to **System Console > Authentication**.

Reviewer: ```suggestion
A one-time, one-click conversion tool is available within the existing **OAuth 2.0** page. You can also go to **System Console > Authentication > OpenID Connect**.
```
